javascript/event-recommender.md

```diff
@@ -5,13 +5,10 @@
 To build a modular system using Object Oriented Programming principles.
 
 ### Overview
-In this project, you'll build an EventRecommender System that will
-help you practice Object Oriented Programming using Javascript. This should take
-a maximum of 3 hours.
+In this project, you'll build an EventRecommender System that will help you practice Object Oriented Programming using Javascript. This should take a maximum of 3 hours.
```

Comment:
`This process should take a maximum of 3 hours` sounds good to me but `This should take a maximum of 3 hours` is also correct.